api/src/main/java/cz/metacentrum/perun/spRegistration/rest/controllers/common/ConfigController.java

```diff
@@ -100,4 +97,14 @@ public boolean getSpecifyAuthoritiesEnabled() {
 		log.trace("getSpecifyAuthoritiesEnabled() returns: {}", specifyAuthoritiesEnabled);
 		return specifyAuthoritiesEnabled;
 	}
+
+	@GetMapping(path = "/api/config/mailsMap")
```

Comment:
Apply rename here as well for both method and mapping

Reply:
Done